### DISCUSSION TITLE
Windows 10 Start Menu

### DISCUSSION THREAD

**Discussion by Jess093** [Open-Shell/Open-Shell-Menu#877] [Ideas]
Good morning everyone

I've been using Open shell for a long time since its inception

But i want to ask a question

Could you add Windows 10 to the menu list?

It is that for me it has more tools and I think it would be good to use it

Put the applications aside without having to look for them and be able to enlarge the screen to add more applications

Yes, I know that you can pin the applications in the other menus to use them
but when you have to use another application
and you have to add something else and edit and accommodate
It becomes like the experience of making access to everything easier becomes slower and more time consuming, and if you do something wrong you have to go back to edit everything to make it accessible

Or at least a skin that is almost the same as the Windows 10 menu

Thanks

**Comment by succdeeznuts** [comment]
There is a Metro and Windows 8 skin they look like a bit of windows 7, 8/8.1/10.
